src/initialization/serverSetup.js

```
@@ -6,13 +6,15 @@ const {
   config: { SERVER_PORT }
 } = require('~/configs/config')
 const scheduledCronJobs = require('~/cron-jobs/scheduledCronJobs')
+const SWAGGER_URL = 'http://localhost:8081/api-docs'
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)